RELEASES.md

```diff
@@ -29,7 +29,8 @@ v4.5.0 (development in progress)
     `decreasing_by` for that function come before the `where`. The
     functions in the `where` clause can have their own termination hints, each following the corresponding definition.
   * The `termination_by` clause must bind exactly those parameters that are not
-    already bound by the function header. If there are none, the `=>` must be
+    already bound by the function header, but by a lambda (e.g. `:= fun x y z =>`)
```

Comment:
What if the lambda is under an `if`?

Reply:
Then it doesn’t count. The behavior is unfortunately rather implementation-defined at the moment, and the real specification is “whatever `lambdaTelescope` does after elaboration”, so I don’t have high hopes for a more helpful specification here than a rough intuition.